metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/stellar/GetProfile.java

```diff
@@ -132,36 +157,26 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(GetProfile.class);
 
-  /**
-   * A client that can retrieve profile values.
-   */
+  // Cached client that can retrieve profile values.
```

Comment:
@nickwallen , I was mistaken.  This IS a private member variable, so the comment can be either format.  I'll change it back, and also change the next one (cachedConfigMap) also, since it's the only one using "//" in this module, even for private members.